data/xml/2021.ijclclp.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2021.ijclclp">
+  <volume id="1" ingest-date="2022-03-28">
+    <meta>
+      <booktitle>International Journal of Computational Linguistics &amp; {C}hinese Language Processing, Volume 26, Number 1, June 2021</booktitle>
+      <editor><first>Chia-Hui</first><last>Chang</last></editor>
+      <editor><first>Berlin</first><last>Chen</last></editor>
+      <publisher>Association for Computational Linguistics and Chinese Language Processing</publisher>
+      <address>Taipei, Taiwan</address>
+      <month>June</month>
+      <year>2021</year>
+    </meta>
+    <frontmatter>
+      <url hash="bad38a08">2021.ijclclp-1.0</url>
+      <bibkey>ijclclp-2021-international</bibkey>
+    </frontmatter>
+    <paper id="1">
+      <title>The <fixed-case>NTNU</fixed-case> <fixed-case>T</fixed-case>aiwanese <fixed-case>ASR</fixed-case> System for <fixed-case>F</fixed-case>ormosa Speech Recognition Challenge 2020</title>
+      <author><first>Fu-An</first><last>Chao</last></author>
+      <author><first>Tien-Hong</first><last>Lo</last></author>
+      <author><first>Shi-Yan</first><last>Weng</last></author>
+      <author><first>Shih-Hsuan</first><last>Chiu</last></author>
+      <author><first>Yao-Ting</first><last>Sung</last></author>
+      <author><first>Berlin</first><last>Chen</last></author>
+      <url hash="62acd771">2021.ijclclp-1.1</url>
+      <bibkey>chao-etal-2021-ntnu</bibkey>
+    </paper>
+    <paper id="2">
+      <title><fixed-case>NSYSU</fixed-case>-<fixed-case>MITL</fixed-case>ab團隊於福爾摩沙語音辨識競賽2020之語音辨識系統 (<fixed-case>NSYSU</fixed-case>-<fixed-case>MITL</fixed-case>ab Speech Recognition System for <fixed-case>F</fixed-case>ormosa Speech Recognition Challenge 2020)</title>
```

Comment:
All papers in non-English languages should have a `<language>` tag (I think it's `zho` here?)